metadata/src/main/java/org/apache/kafka/image/TopicDelta.java

```diff
@@ -118,13 +118,15 @@ public boolean hasPartitionsWithAssignmentChanges() {
 
     /**
      * Find the partitions that have change based on the replica given.
-     *
+     * <p>
      * The changes identified are:
-     *   1. deletes: partitions for which the broker is not a replica anymore
-     *   2. electedLeaders: partitions for which the broker is now a leader (leader epoch bump on the leader)
-     *   3. leaders: partitions for which the isr or replicas change if the broker is a leader (partition epoch bump on the leader)
-     *   4. followers: partitions for which the broker is now a follower or follower with isr or replica updates (partition epoch bump on follower)
-     *
+     *   <ul>1. deletes: partitions for which the broker is not a replica anymore</ul>
```

Comment:
Is it `<ul>` `<li>` ?

ref: https://blog.joda.org/2012/11/javadoc-coding-standards.html